client/src/components/EventView/index.js

```diff
@@ -1,7 +1,174 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import AppBar from "../Common/AppBar";
+import { eventPink, theme1, spacing64, fontSize24 } from "../../resources/style-constants";
+import Invitee from "./Invitee/index.js";
+import Map from "../Common/Map/index.js"
+import styled from "styled-components";
+import Button from "../Common/Buttons/SecondaryButton/index.js";
+import RSVPFloating from "../EventView/RSVPFloating/index.js";
+import ClientService from "../../services";
+import CenteredLoadingSpinner from "../Common/CenteredLoadingSpinner";
+import Stack from "../Common/Stack";
+import Error from "../Common/Error";
+//import PageAccess from "../Common/PageAccess";
 
-const EventView = () => {
-  return <div>Event View</div>;
+// const testEvent = {
+//     eventID: "testID",
+
+// }
+
+const CenterHeading = styled.p`
+    font-size: calc(10px + 2vmin);
+    color: black;
+    font-weight: bold;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+`;
+
+const CenterHeadingLight = styled.p`
+    font-size: calc(7px + 2vmin);
+    color: black;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+`;
+
+const Heading = styled.p`
+    font-size: calc(10px + 2vmin);
+    color: black;
+    font-weight: bold;
+`;
+
+const DisplayStle = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: left;
+    align-items: left;
+    width: 70%;
+    height: 100%;
+    padding: 10% 15%;
+`;
+
+const InviteeRow = styled.div`
+  display: flex;
+  flex-direction: row;
+  //align-items: center;
+  align-items:flex-start;
+  //justify-content: space-around;
+`;
+
+const Main = () => {
+  const [apiStatus, setApiStatus] = useState({
+    loading: false,
+    error: false,
+    message: false
+  });
+
+    const [event, setEvent] = useState({
+        eventName: "testName"
+
+    });
+    useEffect(() => {
+      loadPageData();
+      // eslint-disable-next-line
+    }, []);
+
+    //not sure how to implement this
+    // useEffect(() => {
+    //     const params = new URLSearchParams(event.location.search);
+    //     let eventId = params.get("id");
+    //     if (eventId) {
+    //     loadPageData(eventId);
+    //     } else {
+    //     //redirect to home page since there isn't an eventId
+    //     }
+    //     // eslint-disable-next-line
+    //     }, [location.search]);
+    //please help Morgan!!
+
+  const loadPageData = async () => {
```

Comment:
Then for this, you need to pass the eventId that you got from the url as a parameter to this function